src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -8664,11 +8664,10 @@ private bool IsPossibleExpression()
                     // expression (whether it is used as an identifier or a keyword).
                     return this.IsTrueIdentifier() || (this.CurrentToken.ContextualKind == SyntaxKind.FromKeyword);
                 default:
-                    return IsExpectedPrefixUnaryOperator(tk)
```

Comment:
>IsExpectedPrefixUnaryOperator(tk) [](start = 27, length = 33)

It is not obvious why the check for IsExpectedPrefixUnaryOperator is no longer needed.

Reply:
the call to SyntaxFacts.IsAnyUnaryExpression() already handles it.

---
In reply to: [185140240](https://github.com/dotnet/roslyn/pull/26405#discussion_r185140240) [](ancestors = 185140240)